core/src/main/java/hudson/Functions.java

```diff
@@ -665,7 +627,24 @@ public static boolean isAutoRefresh(HttpServletRequest request) {
     public static boolean isCollapsed(String paneId) {
     	return PaneStatusProperties.forCurrentUser().isCollapsed(paneId);
     }
-    
+
+    public static boolean useUserTimeZone() {
+        return TimeZoneProperty.forCurrentUser() != null;
+    }
+
+    public static String getUserTimeZone() {
```

Comment:
I suggest annotating methods properly so that nullability is explicit

Reply:
added